lib/TopSitesFeed.jsm

```diff
@@ -360,6 +367,32 @@ this.TopSitesFeed = class TopSitesFeed {
     }
   }
 
+  async updateSearchShortcuts() {
+    if (!this.store.getState().Prefs.values[SEARCH_SHORTCUTS_EXPERIMENT]) {
+      return;
+    }
+
+    if (!this._tippyTopProvider.initialized) {
+      await this._tippyTopProvider.init();
+    }
+
+    // Populate the state with available search shortcuts
+    const searchShortcuts = [];
+    SEARCH_SHORTCUTS.forEach(shortcut => {
+      // Only add the shortcut if the engine is available.
+      if (!Services.search.getEngines().find(e => e.identifier && e.identifier.match(shortcut.searchIdentifier))) {
```

Comment:
@Mardak ^ included the guard for `identifier` 😉

Reply:
I think you need to `await new Promise(resolve => Services.search.init(resolve));` here to make sure search is ready here